### DISCUSSION TITLE
Github Campus Registration

### DISCUSSION THREAD

**Discussion by gmsvns** [community/community#133217] [GitHub Education]
### Hi everyone,

I am trying to enroll my institution in GitHub Campus Program, But GitHub Reject our without providing any reason.
I have raised ticket to GitHub Support portal but they did not respond on our query, its more than 2 weeks .

**Comment by praneeth-balamurugan** [comment]
just get the access for your account by just providing the .edu  mail

**Reply by gmsvns** [reply]
we are provided everything to github education team 
like .edu email,  Affiliation Letter, Recognition Certificate,  Self Affidavit, Society Registration Certificate, SARAS
We are provided everything to GitHub Education Team But They are rejected our application, 
Without any reason
